compiler/parser/src/lexer.rs

```diff
@@ -1366,6 +1379,8 @@ mod tests {
         Tok::String {
             value: s.to_owned(),
             kind: StringKind::Normal,
+            prefix: None,
+            triple_quoted: false,
         }
     }
```

Comment:
It might be a good idea to change the definition of this function to accept non-default values for `prefix` and `triple_quoted`.

Reply:
@fanninpm Thanks for the comment. Yeah, I'll add `prefix` and `triple_quoted`.

```rust
    fn stok(s: &str, prefix: Option<String>, triple_quoted: bool) -> Tok {
        Tok::String {
            value: s.to_owned(),
            kind: StringKind::Normal,
            prefix,
            triple_quoted,
        }
    }
```

Reply:
Fixed.